### PR TITLE
Most likely fix Slimefun-Addon-Community/Galactifun#70

### PR DESCRIPTION
**Pull request by Seggan**
## Description

Fix a NPE in Galactifun

## Code Changes

Added a null check before registering item

## Checklist

- [ ] I have also tested the proposed changes with an addon.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [ ] I have added sufficient Unit Tests to cover my code.
- [x] I updated the version in pom.xml according to semantic versioning.

### CODE REVIEW

**Comment by Riley31415** [comment]
Tests failed, also when would either one be null? its only called by SlimefunItem which prevents both things from being null

**Comment by Seggan** [comment]
/shrug biscuit said it was due to us blindly registering recipes with null elements

**Comment by Seggan** [comment]
See https://pastebin.com/zmRgEx5b

File: src/main/java/io/github/mooy1/infinitylib/machines/MachineRecipeType.java [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 import lombok.Getter;
 
+import org.apache.commons.lang.Validate;
```
Comment (Riley31415) [inline src/main/java/io/github/mooy1/infinitylib/machines/MachineRecipeType.java:11]: unused import

**Review by Riley31415 — CHANGES_REQUESTED** [review]

**Pull request merged**